tests/snuba/search/test_backend.py

```diff
@@ -0,0 +1,157 @@
+from __future__ import absolute_import
+
+
+from datetime import datetime, timedelta
+
+from sentry.event_manager import ScoreClause
+from sentry.models import (
+    Environment, GroupAssignee, GroupBookmark, GroupStatus, GroupSubscription,
+)
+from sentry.search.base import ANY
+from sentry.search.django.backend import DjangoSearchBackend
+from sentry.testutils import SnubaTestCase
+
+
+class SnubaSearchTest(SnubaTestCase):
```

Comment:
The general setup/tests are kinda dupes of the existing search tests: https://github.com/getsentry/sentry/blob/86c78d3faf50d1c2167fce526bff57aeb4f34237/tests/sentry/search/django/tests.py#L22

The difference is I turn on the `snuba` project switch, and we know CH+Snuba are running in the `tests/snuba` suite.

So, I don't duplicate all the work here, like verifying sorting, because that's all still done in normal Django code. I just test environment/tags searching pieces.